### PR TITLE
implemented shortest job first operating systems algorithm

### PR DESCRIPTION
**Pull request by vaibhavdaren**

### CODE REVIEW

File: code/operating-systems/fcfs.cpp [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+    for (int  i=0; i<n; i++)
+    {
+        total_wt = total_wt + wt[i];
+        total_tat = total_tat + tat[i];
+        cout << "   " << i+1 << "\t\t" << bt[i] <<"\t    "
+            << wt[i] <<"\t\t  " << tat[i] <<endl;
+    }
+ 
+    cout << "Average waiting time = "
+         << (float)total_wt / (float)n;
```
Comment (arnavb) [inline code/operating-systems/fcfs.cpp]: Use `static_cast`

File: code/operating-systems/fcfs.cpp [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+    cout << "\nAverage turn around time = "
+         << (float)total_tat / (float)n;
+}
+ 
+// Driver code
+int main()
+{
+    //process id's
+    int processes[] = { 1, 2, 3};
```
Comment (arnavb) [inline code/operating-systems/fcfs.cpp]: Use vectors, not raw arrays

**Comment by AdiChat** [comment]
Kindly remove the file `fcfs.cpp` as it has been merged in your previous commit 👍

File: code/operating-systems/fcfs.cpp (hunk 2 above)
Comment (vaibhavdaren) [reply code/operating-systems/fcfs.cpp]: okay

**Comment by vaibhavdaren** [comment]
cool understood

**Pull request closed without merging**